Makefile

```diff
@@ -13,7 +13,7 @@ PG_VERSION_NUM = 170004
 PG_BRANCH = REL_17_STABLE
 PROTOC_VERSION = 25.1
 
-VERSION = 5.1.0
```

Comment:
Note we missed updating this when we released 6.0.0, but because this is mostly not used (only for edge cases where someone wants to build a shared library) we did not notice until now.